src/components/desktop/desktop.vue

```diff
@@ -1,13 +1,22 @@
 <template>
   <FixedHeaderAndAsidesLayout>
     <template #header>
-      <div class="x-list x-list--horizontal x-list__item--expand">
-        <div class="x-list x-list--padding-top x-list--padding-04">
+      <div
+        class="
+          x-list__item--expand
+          x-row x-row--align-start x-row--gap-04
+          x-padding--top-06 x-padding--bottom-07
+        "
+      >
+        <div class="x-row__item x-row__item--span-2 x-list x-padding--top-04">
           <Logo />
         </div>
 
         <div
-          class="x-list x-list--vertical x-list--gap-05 x-list--align-stretch x-list__item--expand"
+          class="
+            x-row__item x-row__item--span-8
+            x-list x-list--vertical x-list--gap-05 x-list--align-stretch
+          "
         >
           <div class="x-relative">
             <SearchBox />
```

Comment:
Can you change the SearchIcon inside the component to be the tiny one:

`<SearchTinyIcon class="x-icon--l" />`